### PR TITLE
Rust demand map lane guide

### LINKED ISSUE #182: Rust docs: reference/demand map lanes

### PR DESCRIPTION
**Pull request by SirCipher**
Blocked by #210 
Closes #182

### CODE REVIEW

File: src/_rust-server/demand-map-lanes.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
+---
+title: Demand Map Lanes
+short-title: Demand Map Lanes
+description: "Remotely command Web Agents to take action, and observe the actions taken by others."
+group: Reference
+layout: documentation
+cookbook: https://github.com/swimos/cookbook/tree/master/demand_value_lanes
```
Comment (PiggottDev) [inline src/_rust-server/demand-map-lanes.md:7]: Think some of these paths should be `demand_map_lanes`

File: src/_rust-server/demand-map-lanes.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+        .await
+        .expect("Failed to listen for ctrl-c.");
+}
+```
+
+Further reading: [Downlinks]({% link _rust-server/downlinks.md %})
+
+# Try It Yourself
+
+A standalone project that demonstrates Command Lanes is available [here](https://github.com/swimos/swim-rust/tree/main/example_apps/demand_map_lane).
```
Comment (PiggottDev) [inline src/_rust-server/demand-map-lanes.md:225]: `Demand Map Lanes`

**Review by PiggottDev — COMMENTED** [review]
Just some minor corrections.

Should mention I'm only reviewing the page corresponding to the PR title.

File: src/_rust-server/demand-map-lanes.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+            .map(|value: Option<Order>| value.map(|order| order.value))
+    }
+}
+```
+
+# Use cases
+
+Demand Map Lanes are suitable for situations where you aren't interested in the data immediately and can handle the delay between linking to the lane and the `cue_key` invocation. Common usecases are:
+
+- Propagating metadata. Metadata may be propagated at set intervals using the [Handler Context's]({% link _rust-server/handler-context.md %}) timer API and calculated on-demand.
```
Comment (horned-sphere) [inline src/_rust-server/demand-map-lanes.md:89]: I'd remove the metadata one again.

**Review by horned-sphere — APPROVED** [review]

**Pull request merged**